.storybook/webpack.config.js

```diff
@@ -1,8 +1,26 @@
 const path = require('path');
 
+const useExperimentalFeatures =
```

Comment:
Instead of modifying webpack config, could we define the feature flags in the scss file that we have?

Reply:
That can definitely be an option. The approach with WebPack config is to allow feature flag switch without modifying code.

Reply:
Definitely, which can totally be useful. I can go either way 👍